component-samples/demo/README.MD

```diff
@@ -151,18 +151,15 @@ If no proxy needs to be specified, do not add these properties to your _JAVA_OPT
 
 # Configuring OnDie (Optional)
 
-OnDie is a type of device that makes use of the MAROE prefix. If you need to support such devices then you will need to configure the FDO PRI demo components by adding/updating the following properties. The values can be specified via Java -Doptions, entries in application.properties or entries in the .env files.
+OnDie is a type of device with device RoT added during manufacturing. To support OnDie devices you will need to import the OnDie certificates into the database via the api on both the Manufacturer and Owner components.
 
-OnDie requires several certificates and CRLs. 
+    - `POST /api/v1/ondie` to upload OnDie certificates into the database.
 
```

Comment:
Copy the csme.zip/csme-debug.zip to Manufacturer & Owner components. Add the below line to Dockerfile/Podmanfile of Manufacturer & Owner :
COPY ./csme.zip ./csme.zip
Restart Docker/Podman containers and the execute CURL command to upload Ondie artifacts to DB